docs/lifecycle.md

```diff
@@ -1,11 +1,11 @@
 # What is a transaction
 
-* A [`Transaction`](https://github.com/fedimint/minimint/blob/563c600287decd47e89e15e29ab478648395f378/minimint-core/src/transaction.rs#L12-L16) Consists of inputs and outputs
-  * Each input and output belongs to a certain module (e.g. Mint, Wallet, …)
-  * All inputs have to add up to the same sum as all outputs
-* Also contains a signature
-  * Each input has an associated secret/public key pair
-  * The signature is a MuSig2 multi signature with all these input keys signing the entire transaction
+- A [`Transaction`](https://github.com/fedimint/fedimint/blob/master/fedimint-core/src/transaction.rs) Consists of inputs and outputs
```

Comment:
This previously was a permanent link. Though considering that CI has markdown link checker, I guess it will not become accidentally stale, so I guess OK?

Reply:
Could we just keep the old link?

Reply:
I'd just update the link while at it, but yes.

Reply:
KK, updated to permalink